README.md

```diff
@@ -20,6 +20,7 @@ addlicense requires go 1.16 or later.
     -f custom license file (no default)
     -l license type: apache, bsd, mit, mpl (defaults to "apache")
     -y year (defaults to current year)
+    -x custom license check, can be used with custom license to properly check presence of header (defaults to "(c)")
```

Comment:
The string "(c)" may result in some false positives, e.g. an `(a) (b) (c)` list or a reference to a 501(c)3 organization.